emuDCS/PeripheralApps/src/common/EmuPeripheralCrateConfig.cc

```diff
@@ -7625,6 +7629,101 @@ void EmuPeripheralCrateConfig::readoutScope(xgi::Input * in, xgi::Output * out )
   //
 }
 //
+void EmuPeripheralCrateConfig::TMBFiberReset(xgi::Input * in, xgi::Output * out )
+  throw (xgi::exception::Exception) {
+  //
+  cgicc::Cgicc cgi(in);
+  //
+  cgicc::form_iterator name = cgi.getElement("tmb");
+  //
+  int tmb=0;
+  if(name != cgi.getElements().end()) {
+    tmb = cgi["tmb"]->getIntegerValue();
+    std::cout << "TMB " << tmb << std::endl;
+    TMB_ = tmb;
+  }
+  //
+	TMB * thisTMB = tmbVector[tmb];
+  //
+  name = cgi.getElement("mode");
+	//
+	std::string mode = "";
```

Comment:
4-char tabs instead of 8 would be nicer as well so that nested scopes don't run away too far to the right